nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -265,12 +460,27 @@ def _permuted_ols_on_chunk(scores_original_data, tested_vars, target_vars, threa
                     "(%0.2f%%, %i seconds remaining)%s"
                     % (thread_id, i, n_perm_chunk, percent, remaining, crlf))
 
-    return scores_as_ranks_part, h0_fmax_part.T
-
-
-def permuted_ols(tested_vars, target_vars, confounding_vars=None,
-                 model_intercept=True, n_perm=10000, two_sided_test=True,
-                 random_state=None, n_jobs=1, verbose=0):
+    return (
+        vfwe_scores_as_ranks_part,
+        h0_vfwe_part.T,
+        h0_csfwe_part.T,
+        h0_cmfwe_part.T,
+    )
+
+
+def permuted_ols(
+    tested_vars,
+    target_vars,
+    confounding_vars=None,
+    model_intercept=True,
+    masker=None,
+    cdt=1.,
```

Comment:
I think I'd prefer to have the cluster-defining threshold be a p-value instead of a t-statistic. Having a default t-statistic isn't very useful, given that the appropriate value depends on the degrees of freedom.

Reply:
The default p-value threshold would, of course, be 0.001.

Reply:
I replaced `cdt` with a p-value `threshold` parameter. I need to take a second look at the calculation of the minimum t-statistic though.